engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -506,9 +512,12 @@ class WorkflowActor(workflowToStart: WorkflowToStart,
       }
   }
 
-  private def filesNotFoundErrorMsg(files: List[Path]) = {
-    if(files.nonEmpty) s" File(s) not found during deletion: ${files.mkString(",")}"
+  private def deleteFilesAdditionalError(filesNotFound: List[Path], callCacheInvalidationErrors: List[Throwable]): String = {
+    val filesNotFoundMsg = if(filesNotFound.nonEmpty) s" File(s) not found during deletion: ${filesNotFound.mkString(",")}" else ""
+    val invalidationErrorMsg = if(callCacheInvalidationErrors.nonEmpty) s" Call cache invalidation error(s): ${callCacheInvalidationErrors.map(ExceptionUtils.getMessage)}"
     else ""
```

Comment:
nitpicky: this `else` feels weird here on this line. Maybe it'd make sense to change formatting a bit.